### PR TITLE
Installed node-sass@5.0.0. and .scss files for styling

### LINKED ISSUE #14: Styling and rendering home page and adding scss package
**Is your feature request related to a problem? Please describe.**
Structuring and styling home page and adding sass package.

**Describe the solution you'd like**
Modulizing styling using scss and rendering the home page.

### PR DESCRIPTION
**Pull request by connielion**
## Changes
1. Install node-sass and add sass.
2.Added logo image into public directory/ 

## Purpose
Style NavBar and Searched.

## Approach
Used sass to modularize styling.

Closes #14

### CODE REVIEW

**Comment by Lap343** [comment]
You guys are fast! So much code done !

File: movie-app/src/body/Searched.js [hunk 1]
```diff
@@ -4,14 +4,14 @@ const Searched = ({ movies }) => {
     const moviesCheck = () => {
         if (movies){
             return movies.map(movie => {
-                return <MovieCard key={movie.imdbID} movie={movie}/> 
+                return <MovieCard key={movie.imdbID} movie={movie}/>
```
Comment (kmausolf) [inline movie-app/src/body/Searched.js:7]: SMART!  If you key it by the `imdbID`, you get the added bonus of their database's primary key uniqueness invariant.

File: movie-app/src/App.js [hunk 2]
```diff
@@ -12,10 +15,11 @@ function App() {
   },[])
 
   return (
-    <div>
+    <div className="App">
```
Comment (kmausolf) [inline movie-app/src/App.js:18]: Try to keep css class names kebab-case:  ex: `app-main-container` and `kebab-case`

File: movie-app/src/scss/components/body/_search_results.scss [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+}
+
+@import "./_card";
+
+/*
+display: grid;
+    grid-template-columns: repeat(auto-fill, minmax(29rem, 1fr));
+    grid-column-gap: 10px;
+    grid-row-gap: 10px;
+*/
```
Comment (kmausolf) [inline movie-app/src/scss/components/body/_search_results.scss:17]: Is this css needed?

**Review by kmausolf — COMMENTED** [review]
PR seems decently focused on solving the issue presented.  Code and structure looks pretty good, so nice work on that!  Only reason I'm not approving is that I'd like an apprentice reviewer to conduct the official review.

**Review by Lap343 — APPROVED** [review]
Great job team! The code looks good to me.

**Review by krferrell — APPROVED** [review]
Looking good team.

**Pull request merged**